FitGoal/FitGoal/Singup&Singin/CustomAuthenticator.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  CustomAuthenticator.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 10/4/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import Foundation
+
+class CustomAuthenticator {
```

Comment:
Could you please explain what's the purpose of this class. Currently its name implies that it's an authenticator, but it doesn't authenticate.